<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseMoveBackupFolderErrorDialog</class>
 <widget class="QDialog" name="ChooseMoveBackupFolderErrorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>325</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Stop backup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*ColorTokenIcon;Button;bConfirm;normal;off;colorToken.text-inverse-accent*/
/*ColorTokenIcon;Button;bConfirm;disabled;off;colorToken.text-inverse-accent*/
/*ColorTokenIcon;Button;bConfirm;active;off;colorToken.text-inverse-accent*/
/*ColorTokenIcon;Button;bConfirm;selected;off;colorToken.text-inverse-accent*/

#ChooseMoveBackupFolderErrorDialog
{
background-color: #ffffff; /*colorToken.page-background*/
}

</string>
  </property>
  <property name="TOKENIZED" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
   <property name="leftMargin">
    <number>24</number>
   </property>
   <property name="topMargin">
    <number>24</number>
   </property>
   <property name="rightMargin">
    <number>24</number>
   </property>
   <property name="bottomMargin">
    <number>31</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
     <property name="spacing">
      <number>24</number>
     </property>
     <item>
      <widget class="QLabel" name="labelFolderIcon">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>96</width>
         <height>96</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../../gui/Resources_common.qrc">:/images/alert-triangle-small.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="lTitle">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Error moving remote backup folder</string>
         </property>
         <property name="dimension" stdset="0">
          <string notr="true">label-2</string>
         </property>
         <property name="semibold" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignTop">
        <widget class="QLabel" name="lDescription">
         <property name="text">
          <string>Failed to move or remove the remote backup folder. Reason: destination folder already exist</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <property name="font-size" stdset="0">
          <string notr="true">body-2</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="moveToContainer" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>69</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>21</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lMoveTo">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Move to:</string>
        </property>
        <property name="semibold" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>8</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lTarget">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="type" stdset="0">
           <string notr="true">mega</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bChange">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>Change</string>
          </property>
          <property name="type" stdset="0">
           <string notr="true">secondary</string>
          </property>
          <property name="dimension" stdset="0">
           <string notr="true">medium</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bCancel">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="type" stdset="0">
        <string notr="true">outline</string>
       </property>
       <property name="dimension" stdset="0">
        <string notr="true">medium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bConfirm">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>36</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>36</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Move Backup</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <property name="type" stdset="0">
        <string notr="true">primary</string>
       </property>
       <property name="dimension" stdset="0">
        <string notr="true">medium</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../gui/Resources_common.qrc"/>
 </resources>
 <connections/>
</ui>
